core/frontend.ml

```diff
@@ -75,6 +75,18 @@ struct
         program), ffi_files)
 
 
+let program tyenv pos_context program =
```

Comment:
Is there any way to have this for REPL `sentence`s as well?

Reply:
yep, done :)